descwl_shear_sims/simple_sim.py

```diff
@@ -252,8 +252,8 @@ def __init__(
         self.psf_kws = copy.deepcopy(PSF_KWS_DEFAULTS[self.psf_type])
         if psf_kws is not None:
             self.psf_kws.update(psf_kws)
-        self.psf_g1 = self.psf_kws.pop('psf_g1', 0.0)
-        self.psf_g2 = self.psf_kws.pop('psf_g2', 0.0)
+        self.psf_g1 = self.psf_kws.pop('g1', 0.0)
+        self.psf_g2 = self.psf_kws.pop('g2', 0.0)
```

Comment:
My final comment is that we should probably just store the whole dictionary and then pull out what we need in the method, but I am willing to pass on that.

Reply:
The issue there is that we need to pop these because they are not allowed keywords to the Gaussian itself.

Reply:
So we would need to do this in the method to avoid popping the main dictionary
```
kw = deepcopy(self.psf_kws)
g1 = kw.pop('g1', 0.0)
g2 = kw.pop('g2', 0.0)
```
which would be fine too

Reply:
Right. Eventually, we can use the galsim parsing internals to build this automatically with any allowed keywords in a galsim config. However, let's merge for now and punt on that.